data/circulation_policies.json

```diff
@@ -16,8 +16,8 @@
   },
   {
     "$schema": "http://ils.rero.ch/schema/circ_policies/circ_policy-v0.0.1.json",
-    "name": "library: short",
-    "description": "library: short loan (7 days)",
+    "name": "Site d'Aoste: short",
```

Comment:
if a librarian adds this policy to another library, then the policy name is no longer valid.

Reply:
Ok. We need the input of the sleeping Consortium WG on this.